modules/central-services-public-config.adoc

```diff
@@ -90,6 +90,12 @@ include::snippets/technology-preview.adoc[]
 |===
 | Parameter | Description
 
+|`central.declarativeConfiguration.mounts.configMaps`
+| Mounts a config map used for declarative configurations.
```

Comment:
Not sure how the typical language is for cases like this, but right now this setting allows to mount a _list_ of config maps used for declarative config, so potentially better to do the plural here (same applies to the secrets below).